add logdiff.sh tool to help analyze test logs between runs.

domdom82 opened the pull request:
This small tool should allow us to remove always changing information from log files.
It can be used to easily diff two log files from different test runs. The diff should then show only significant differences.

ioana-blue commented:
This seems harmless. Could you please rebase?

rabbah commented:
Note that orthogonal changes don't need to be rebased by the contributor (it's nice but necessary as this can be done as part of accepting the pull request).

rabbah commented:
@domdom82 can you elaborate the instructions to show how to use this more fully - specifically, how does one get the activation logs to run this script.

domdom82 commented:
I have added some more detailed info on how to get logs.

We usually copy out logs from a previous stable Jenkins build and a failed build after we detect errors. The readme describes how to do it manually using wskadmin.

Also rebased and squashed.

rabbah commented:
Excellent! Thanks.
